ynput/ayon-unreal
# Remove look product type creator

## Changelog Description
Remove the look product type creator


## Additional info
n/a


## Testing notes:

1. Launch Unreal
2. Ayon -> Publisher
3. Publish anything
4. Should be working smoothly
